src/test/java/com/google/firebase/remoteconfig/ParameterTest.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.firebase.remoteconfig;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Map;
+
+import org.junit.Test;
+
+public class ParameterTest {
+
+  @Test
+  public void testConstructor() {
+    final Parameter p = new Parameter();
+    assertNotNull(p.getConditionalValues());
+    assertEquals(0, p.getConditionalValues().size());
+    assertNull(p.getDefaultValue());
+    assertNull(p.getDescription());
+  }
+
+  @Test(expected = NullPointerException.class)
+  public void testConstructorWithNullParameterResponse() {
+    Parameter p = new Parameter(null);
```

Comment:
Just `new Parameter(null)` without the assignment

Reply:
Good catch! Thanks!
Updated.